databricks/koalas/tests/test_dataframe.py

```diff
@@ -4019,3 +4019,60 @@ def test_from_dict(self):
         pdf = pd.DataFrame.from_dict(data, orient="index", columns=["A", "B", "C", "D"])
         kdf = ks.DataFrame.from_dict(data, orient="index", columns=["A", "B", "C", "D"])
         self.assert_eq(pdf, kdf)
+
+    def test_lookup(self):
+        pdf = pd.DataFrame(
+            {
+                "A": [3, 4, 5, 6, 7],
+                "B": [10.0, 20.0, 30.0, 40.0, 50.0],
+                "C": ["a", "b", "c", "d", "e"],
+            }
+        )
+        kdf = ks.from_pandas(pdf)
```

Comment:
What if the index or the column is duplicated?

Reply:
It should raise `ValueError: DataFrame.lookup requires unique index and columns`.
I'll address it. Thanks :)